### PR TITLE
Features/shelters/story 29

### PR DESCRIPTION
**Pull request by jordanholtkamp**
**Functionality:** Add dynamic flash messages for update and new actions

**Testing:**All tests passing and behaving in devo.

**Related Issues:**
#28

### CODE REVIEW

File: app/controllers/shelters_controller.rb [hunk 1]
```diff
@@ -11,9 +11,13 @@ def show
...
 
   def create
-    shelter = Shelter.create(shelter_params)
-
-    redirect_to '/shelters'
+    shelter = Shelter.new(shelter_params)
+    if shelter.save
+      redirect_to '/shelters'
+    else
+      flash[:error] = shelter.errors.full_messages.to_sentence
```
Comment (mintona) [inline app/controllers/shelters_controller.rb]: Niceeee!

File: app/controllers/shelters_controller.rb [hunk 2]
```diff
@@ -23,7 +27,12 @@ def edit
   def update
     shelter = Shelter.find(params[:id])
     shelter.update(shelter_params)
-    redirect_to "/shelters/#{shelter.id}"
+    if shelter.save
+      redirect_to "/shelters/#{shelter.id}"
+    else
+      flash[:error] = shelter.errors.full_messages.to_sentence
+      redirect_back(fallback_location: '/shelters')
```
Comment (mintona) [inline app/controllers/shelters_controller.rb]: I think this can actually be `render: edit` vs having a redirect_back

Comment (jordanholtkamp) [reply app/controllers/shelters_controller.rb]: I tried render edit and it was not working for me. We can look in the am

**Comment by jordanholtkamp** [comment]
Updating

**Pull request closed without merging**